code/modules/hivemind/mobs.dm

```diff
@@ -0,0 +1,913 @@
+//Default speaking chance, can take pilot and speak with him
+//Very rarely can appears from infested machinery
+//////////////////////////////////////////////////////////////////////////////
+
+/mob/living/simple_animal/hostile/hivemind/mechiver
+	name = "RIPley"
+	desc = "Good looking shining new ripley. But wait, there something wrong..."
+	icon = 'icons/mob/hivemind.dmi'
+	icon_state = "mechiver-closed"
+	icon_dead = "mechiver-dead"
+	health = 450
+	maxHealth = 450
+	melee_damage_lower = 10
+	melee_damage_upper = 15
+	mob_size = MOB_LARGE
+	attacktext = "tramples"
+	ability_cooldown = 1 MINUTE
+	speak_chance = 5
+	speed = 16
+	//internals
+	var/pilot						//Yes, there's no pilot, so we just use var
+	var/mob/living/passenger
+	var/hatch_closed = TRUE
+	//default speaking
+	speak = list(
+				"Somebody, just tell him shut up...",
+				"Bzew-zew-zewt. Th-this way!",
+				"Wha-a-at? When i'm nearby this cargo, i feel... fe-fe-fea-fear-er.")
+	target_speak = list(
+				"Come here, jo-jo-join me. Join us-s.",
+				"Time to be-to be-to be whole.",
+				"Enter me, i'm be-best mech among all of these rusty buckets.",
+				"I'm dying. I can't see my ha-hands! I'm scared, hu-hu-hug me.",
+				"I'm not done, it can't be... Hey! Hey you, enter me!")
+	//speaking with pilot
+	var/list/common_answers = list(
+								"Right, chief.",
+								"Yes.",
+								"Right.",
+								"True.",
+								"Yep.",
+								"That's right, chief.")
+	var/list/other_answers = list(
+								"Pathetic.",
+								"How curious.",
+								"We can use it.",
+								"Useless.",
+								"Disgusting")
+	//pilot quotes
+	var/list/pilot_target_speak = list(
+						"Hey! Hey you, wanna, hah, ri-i-ide? It's free!",
+						"Look at this one! Let's-s-s... Take it.",
+						"Wait a minute, we just want to fu-fu-fun with you!",
+						"I see you. We see you.",
+						"Get in! I've give a seat for you.",
+						"Don't be afraid, it's almost painless.")
+	var/list/pilot_commontalk = list(
+						"They are so unfinished, so fragile-ile.",
+						"Look at these... Creatures, i'll never seen them before.",
+						"Hah, did you hear that? They trying to use some kind of we-wep-weapons!",
+						"Useless things, i'm not satisfied.",
+						"This place sucks, man. So creep-p-pew-wepy and no fun, only rudimentary creatures would be live here.")
+
+
+/mob/living/simple_animal/hostile/hivemind/mechiver/Life()
+	. = ..()
+	update_icon()
+
+	//when we have passenger, we torture him
+	if(passenger && prob(15))
+		passenger.apply_damage(rand(5, 10), pick(BRUTE, BURN, TOX))
+		passenger << SPAN_DANGER(pick(
+								"Something grabs your neck!", "You hear whisper: \" It's okay, now you sa-sa-safe! \"",
+								"You've been hit by something metal", "You almost didn't feel your leg!", "Something liquid covers you!",
+								"You feel awful, cadaveric smell", "Something sharp cut your cheek!",
+								"You feel how worm-like thing trying to get inside your head through ear..."))
```

Comment:
You feel something worm-like trying to wriggle into your skull through your ear.